docs/en/operations/table_engines/mergetree.md

```diff
@@ -315,12 +316,42 @@ For concurrent table access, we use multi-versioning. In other words, when a tab
 Reading from a table is automatically parallelized.
 
 
-## TTL for columns and tables
+## TTL for columns and tables {#table_engine-mergetree-ttl}
 
-Data with expired TTL is removed while executing merges.
+Determines the lifetime of values.
 
-If TTL is set for column, when it expires, value will be replaced by default. If all values in columns were zeroed in part, data for this column will be deleted from disk for part. You are not allowed to set TTL for all key columns. If TTL is set for table, when it expires, row will be deleted.
+The `TTL` clause can be set for the whole table and for each individual column. If `TTL` is set for the whole table, individual `TTL` for columns are ignored.
 
-When TTL expires on some value or row in part, extra merge will be executed. To control frequency of merges with TTL you can set `merge_with_ttl_timeout`. If it is too low, many extra merges and lack of regular merges can reduce the perfomance.  
+
+The table must have the column of the [Date](../../data_types/date.md) or [DateTime](../../data_types/datetime.md) data type. This date column should be used in the `TTL` clause. You can only set lifetime of the data as an interval from the date column value.
+
+```
+TTL date_time + interval
+```
+
+You can set the `interval` by any expression, returning the value of the `DateTime` data type. For example, you can use the `INTERVAL` operator and [time interval](../../query_language/operators.md#operators-datetime) operators.
```

Comment:
"INTERVAL operator and time interval operators" sounds weird, why it's not considered to be the same set of operators?